src/pages/Vai/MintVai/getReadableFeeVai.ts

```diff
@@ -0,0 +1,21 @@
+import BigNumber from 'bignumber.js';
+import { convertWeiToTokens } from 'utilities';
+
+import { VAI_ID } from '../constants';
+
+const getReadableFeeVai = ({
+  valueWei,
+  mintFeePercentage,
+}: {
+  valueWei: BigNumber;
+  mintFeePercentage: number;
+}) => {
+  const feeWei = new BigNumber(valueWei || 0).multipliedBy(mintFeePercentage).dividedBy(100);
```

Comment:
The 0 could be set as a default value in the destruction of the argument. It would make this line a bit more readable

Reply:
Actually if the type is always BigNumber, this should never default to 0 because BigNumber would always be truthy even if it isn't valid

Reply:
Good point yes, I took this code from what we currently have in production without checking it since it's test covered. I'll update this